pages/resetbytoken.php

```diff
@@ -225,10 +247,15 @@
     'logo' => $logo,
     'dependency_check_results' => $dependency_check_results,
 
+    'type' => $type,
+    'random' => $randomString,
+    'random2' => $randomString2,
+    'svgQRCode' => $svgQRCode,
+
     'use_questions' => $use_questions,
     'use_tokens' => $use_tokens,
     'use_sms' => $use_sms,
     'change_sshkey' => $change_sshkey,
+    'use_totp' => $use_totp,
     'action' => $action,
-    'source' => $source,
```

Comment:
I think there is a problem with the menu, it showed me multiple highlighted links.